### ISSUE TITLE
Unable To Feed CSVs to RMD

### ISSUE THREAD

**Issue by theclassycaveman** [shadowmoose/RedditDownloader#220]
### Describe the bug
I tried to follow the instructions, such as obtaining the CSVs and using the command prompt script provided:

-  python redditdownloader --import_csv="path/to/file.csv" [--full_csv]

But it keeps saying that python.exe: can't find '__ main __' module in C:\\users\username\redditdownloader'
I'm not sure where to go from here. 

### Environment Info
 - **Windows 10**
 - **RMD Version: 3.1.0**

**Comment by shadowmoose** [comment]
Hello, thanks for opening the report here. Just to be clear, are you running that command without the brackets around `--full_csv`?

**Comment by theclassycaveman** [comment]
> Hello, thanks for opening the report here. Just to be clear, are you running that command without the brackets around `--full_csv`?

Yes. I've tried it with that command included and without. Still the same issue.

**Comment by shadowmoose** [comment]
Okay, interesting. And you're running this command from within the directory above `/redditdownloader/`?

**Comment by theclassycaveman** [comment]
> Okay, interesting. And you're running this command from within the directory above `/redditdownloader/`?

Simply hold ctrl-shift and right click to open powershell window in the folder with redditdownloader, correct? I've tried it on multiple levels. Within the folder, above the folder, below. Nothing seems to work.

**Comment by shadowmoose** [comment]
If you're running it with that command, it'd be the directory above `/redditdownloader/`. That said, this is a difficult issue to debug without having the environment in front of me. Have you tried using the prebuilt binary releases?
